src-vue/stores/wallets.ts

```diff
@@ -123,24 +130,31 @@ export const useWallets = defineStore('wallets', () => {
       try {
         await loadBalances();
         await Promise.all([stats.isLoadedPromise, currency.isLoadedPromise]);
-        isLoadedResolve();
```

Comment:
Why did you remove isLoadedResolve? The isLoadedPromise that is exported from this file is now never resolved.